### DISCUSSION TITLE
Allow Salesforce inspector App with API Access Control

### DISCUSSION THREAD

**Discussion by rlaunay-mtx** [tprouvot/Salesforce-Inspector-reloaded#432] [Q&A] [answered]
Hello,
Is it possible to whilelist Salesforce inspector app using the API access control settings in salesforce ?
we want to restrict who can use API and whitelist specific apps for specifics users.
I've seen the Consumer Key and Rest Header setting in the Salesforce inspector config but i'm unsure about how to use it ?
I tried creating a connected app in salesforce and use the consumer key from it but no luck so far .

**Comment by tprouvot** [comment, marked as answer]
Hi @rlaunay-mtx,
Yes it is possible and we recommend to do so !

Have you checked this [how-to article](https://tprouvot.github.io/Salesforce-Inspector-reloaded/how-to/#use-sf-inspector-with-a-connected-app) ?
Basically you can create your own connected app or use Salesforce Inspector reloaded's default one.

**Reply by rlaunay-mtx** [reply]
Thanks @tprouvot 
Sorry i missed that doc
